sirepo/auth/__init__.py

```diff
@@ -219,22 +219,34 @@ def cookie_cleaner(self, values):
         values[_COOKIE_STATE] = _STATE_LOGGED_OUT
         return values
 
-    def create_user(self, module, want_login=False):
+    def create_user(self, module_or_method, want_login=False):
```

Comment:
Looks to me that module_or_method is only ever a method

Reply:
login is a bit goofy. It said "method" but was a module so I renamed a var and took create_user private. login needs to be broken up. 695c9ef3f83ac271fecd1f4f45f84129ddb26239